docs/network protocol.md

```diff
@@ -325,7 +325,11 @@ Sent to server to inform it of locations that the client has checked. Used to in
 | locations | list\[int\] | The ids of the locations checked by the client. May contain any number of checks, even ones sent before; duplicates do not cause issues with the Archipelago server. |
 
 ### LocationScouts
-Sent to the server to inform it of locations the client has seen, but not checked. Useful in cases in which the item may appear in the game world, such as 'ledge items' in A Link to the Past. The server will always respond with a [LocationInfo](#LocationInfo) packet with the items located in the scouted location.
+Sent to the server to retrieve the items that are on a specified list of locations. The server will respond with a [LocationInfo](#LocationInfo) packet with the items located in the scouted locations.
+Fully remote clients without a patch file may use this to "place" items onto the in-game locations, such as displaying their names, items classification etc. before/upon pickup.
```

Comment:
Honestly, you could change "Fully remote clients without a patch file" to just "Clients" but that's up to how specific you want to be.
"the" -> "their" for readability, and I think changing "such as" to a common, finite list is a better wording here, especially because the "etc." is _not_ the end of the sentence or in a parenthetical
```suggestion
Fully remote clients without a patch file may use this to "place" items onto their in-game locations, most commonly to display their names or item classifications before/upon pickup.
```